### ISSUE TITLE
getting error while creating word wise and sending to android

### ISSUE THREAD

**Issue by avijitb129** [xxyzz/WordDumb#86]
### Checkboxes

- [X] I have read the [README](https://github.com/xxyzz/WordDumb#how-to-use) file.
- [X] I have not found similar issue or disscussion at GitHub.

### Describe the bug

getting error 

### System Information

windows 10
calibre 6.80 64bit
python 3.10
plugin - 3.26.1

### Error message

```text
calibre, version 6.8.0
ERROR: Unhandled exception: <b>TypeError</b>:SendFile.push_files_to_android() missing 1 required positional argument: 'package_name'

calibre 6.8  embedded-python: True
Windows-10-10.0.19043-SP0 Windows ('64bit', 'WindowsPE')
('Windows', '10', '10.0.19043')
Python 3.10.1
Windows: ('10', '10.0.19043', 'SP0', 'Multiprocessor Free')
Interface language: None
Successfully initialized third party plugins: Gather KFX-ZIP (from KFX Input) (1, 49, 0) && Package KFX (from KFX Input) (1, 49, 0) && Count Pages (1, 12, 0) && Goodreads (1, 7, 1) && Goodreads Sync (1, 16, 2) && KFX metadata reader (from KFX Input) (1, 49, 0) && KFX Input (1, 49, 0) && Set KFX metadata (from KFX Output) (1, 64, 0) && KFX Output (1, 64, 0) && Quality Check (1, 13, 2) && WordDumb (3, 25, 0)
Traceback (most recent call last):
  File "calibre\gui2\__init__.py", line 720, in dispatch
  File "calibre_plugins.worddumb.ui", line 142, in done
  File "calibre_plugins.worddumb.send_file", line 43, in send_files
TypeError: SendFile.push_files_to_android() missing 1 required positional argument: 'package_name'
```

### Reproduce steps

1. connect android via adb and allow file transfer
2. select a book
3. click on create word wise

**Comment by avijitb129** [comment]
note: my phone does't have rooted debugging option

**Comment by avijitb129** [comment]
> You're using an older version 3.25.0, the error you posted was fixed in 3.26.0, update to the latest version will solve this error.
> 
> Only sending Word Wise file requires root permission, you can still use this plugin to send X-Ray file or move the generated Word Wise file manually if your phone is rooted.
> 
> Rooted debugging is only enabled on eng and userdebug build ROM like [LineageOS](https://lineageos.org/).

so, i must have rooted debugging for this to work?

**Comment by xxyzz** [comment]
I don't own Kindle fire device so I don't know whether it requires root or not. I only know the Kindle Android app requires root permission to move Word Wise file.

Goodreads probably needs a valid ASIN metadata in the book file. Currently this plugin adds a random invalid ASIN to the book if the book doesn't have one.
